LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2016 Pelagicore
+Copyright (c) 2016-2017 Pelagicore
```

Comment:
Copyright is for the life of the author, plus 70 years. So it is not really necessary to have 2016 - 2017. The original year of copyright is sufficient.